pip_audit/dependency_source/resolvelib/pypi_provider.py

```diff
@@ -75,8 +87,68 @@ def dependencies(self):
             self._dependencies = list(self._get_dependencies())
         return self._dependencies
 
+    def _get_metadata_for_wheel(self):
```

Comment:
Not really any code changes here apart from the `update_state` lines. I just moved them to methods of the `Candidate` since I'm using the package information in the feedback messages.